chains/proposal.go

```diff
@@ -153,9 +141,9 @@ type TransferMessageData struct {
 	Payload      []interface{}
 }
 
-func NewMessage(source, destination uint8, data interface{}, msgType coreMessage.MessageType) *coreMessage.Message {
+func NewMessage(source, destination uint8, data interface{}, msgType message.MessageType) *message.Message {
```

Comment:
Isn't duplicated with core?

Reply:
It is but that change was made in the next (and the last) PR